.mailmap

```diff
@@ -1,20 +1,89 @@
+
+Borys Jurgiel <info@borysjurgiel.pl> borys <info@borysjurgiel.pl>
+Borys Jurgiel <info@borysjurgiel.pl> <info at borysjurgiel dot pl>
+
+Clemens Raffler <clemens.raffler@gmail.com> <root676@users.noreply.github.com>
+Clemens Raffler <clemens.raffler@gmail.com> root676 <clemens.raffler@gmail.com>
+Clemens Raffler <clemens.raffler@gmail.com> root676 <clemens.raffler@gmx.at>
+
 D'Hont René-Luc <rldhont@gmail.com>
 
+Damiano Lombardi <damiano@opengis.ch> Damiano <damiano@opengis.ch>
+Damiano Lombardi <damiano@opengis.ch> Damiano <lombardi.damiano@gmail.com>
+Damiano Lombardi <damiano@opengis.ch> <lombardi@customcut.ch>
+Damiano Lombardi <damiano@opengis.ch> <lombardi.damiano@gmail.com>
+Damiano Lombardi <damiano@opengis.ch> domi <lombardi@customcut.ch>
+
+David Marteau <dmarteau@3liz.com> <david@innophi.com>
+David Marteau <dmarteau@3liz.com> <dhmarteau@gmail.com>
+
+David Signer <david@opengis.ch> David <david@opengis.ch>
+David Signer <david@opengis.ch> signedav <david@opengis.ch>
+
+Etienne Trimaille <etienne.trimaille@gmail.com> <etienne@Etiennes-MacBook-Pro.local>
+Etienne Trimaille <etienne.trimaille@gmail.com> <gustrimaille@yahoo.fr>
+Etienne Trimaille <etienne.trimaille@gmail.com> Étienne Trimaille <gustrimaille@yahoo.fr>
+
 Even Rouault <even.rouault@spatialys.com> <even.rouault@mines-paris.org>
 
+Francisco Raga <All4Gis@users.noreply.github.com> Fran Raga <All4Gis@users.noreply.github.com>
+
 Gary Sherman <gsherman@geoapt.com> <gsherman@c8812cc2-4d05-0410-92ff-de0c093fc19c>
 
+Germán Carrillo <gcarrillo@linuxmail.org> gacarrillor <gcarrillo@linuxmail.org>
+Germán Carrillo <gcarrillo@linuxmail.org> gacarrillor <geotux_tuxman@linuxmail.org>
+Germán Carrillo <gcarrillo@linuxmail.org> <geotux_tuxman@linuxmail.org>
+Germán Carrillo <gcarrillo@linuxmail.org> Germán <geotux_tuxman@linuxmail.org>
+Germán Carrillo <gcarrillo@linuxmail.org> Germap <gcarrillo@linuxmail.org>
+
+Giovanni Allegri <giovanni.allegri@gmail.com> giohappy <giohappy@gmail.com>
+
+Giovanni Manghi <giovanni.manghi@naturalgis.pt> gioman <giovanni.manghi@faunalia.pt>
+Giovanni Manghi <giovanni.manghi@naturalgis.pt> <gio@sibirica.(none)>
+Giovanni Manghi <giovanni.manghi@naturalgis.pt> <giovanni.manghi@faunalia.pt>
+Giovanni Manghi <giovanni.manghi@naturalgis.pt> Giovanni Manghi <giovanni@sibirica.(none)>
+
 Giuseppe Sucameli <brush.tyler@gmail.com> <brushtyler@881b9c09-3ef8-f3c2-ec3d-21d735c97f4d>
 Giuseppe Sucameli <brush.tyler@gmail.com> <brushtyler@c8812cc2-4d05-0410-92ff-de0c093fc19c>
 
+Harrissou Sant-anna <delazj@gmail.com> DelazJ <delazj@gmail.com>
+
+Ivan Ivanov <ivan.ivanov@suricactus.com> <suricactus@users.noreply.github.com>
+
+Jacky Volpes <jacky.volpes@oslandia.com> <34267385+Djedouas@users.noreply.github.com>
+
+Jean-Roc Morreale <jr.morreale@enoreth.net> Jean Roc <jr.morreale@enoreth.net>
+Jean-Roc Morreale <jr.morreale@enoreth.net> MORREALE Jean-Roc <jr.morreale@enoreth.net>
```

Comment:
hi, the email address is the same, you can merge _MORREALE Jean-Roc_ and _Jean Roc_ with _Jean-Roc Morreale_

Reply:
Right, the proposed mapping will merge all commits made by `Jean Roc <jr.morreale@enoreth.net>` and `MORREALE Jean-Roc <jr.morreale@enoreth.net>` into the preferred name/e-mail pair: `Jean-Roc Morreale <jr.morreale@enoreth.net>`.